### PR TITLE
ARC-210 Buy AR Screen (UI with mock data - pre api)

### PR DESCRIPTION
**Pull request by elehma4**
This includes the basic state management logic as well as the styling/display for the buy screen. In this PR, mock data is used. In the next PR, we will be using the Onramper API for real data.

### CODE REVIEW

File: src/routes/popup/purchase.tsx
Comment (7i7o) [inline src/routes/popup/purchase.tsx:1]: It's perfect in dark mode, now we only need to add light mode 👍

**Review by 7i7o — CHANGES_REQUESTED** [review]
We only need to add light mode

File: src/components/popup/home/BuyButton.tsx [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+  logo
+}: ButtonWrapperProps) {
+  const [push] = useHistory();
+
+  const targetRoute = route === "/purchase" ? "/purchase" : "/confirm-purchase";
+
   return (
-    <ButtonWrapper padding={padding}>
-      <CustomButton className="normal-font-weight" small fullWidth>
+    <ButtonWrapper padding={padding} route={route} logo={logo}>
```
Comment (nicholaswma) [inline src/components/popup/home/BuyButton.tsx:23]: does `logo={logo}` do anything here?

**Review by nicholaswma — APPROVED** [review]
Outside of the light changes that Matias mentioned lgtm!

File: src/components/popup/home/BuyButton.tsx (hunk 1 above)
Comment (elehma4) [reply src/components/popup/home/BuyButton.tsx:23]: the logo prop controls whether we show the arweave logo next to buy now or not @nicholaswma

**Comment by elehma4** [comment]
Closing branch. This branch is now included within https://github.com/arconnectio/ArConnect/pull/157 (ARC-210-buy-ar-screen-live).

**Pull request closed without merging**